drivers/pwm/pwm_nrf5_sw.c

```diff
@@ -85,6 +85,7 @@ static int pwm_nrf5_sw_pin_set(struct device *dev, u32_t pwm,
 	u8_t ppi_index;
 	u8_t channel;
 	u16_t div;
+	u32_t initpol;
```

Comment:
Move up in the declarations, so as to be sorted in decreasing line lengths.

Reply:
Fixed.